tmva/tmva/src/DNN/Architectures/Reference/Dropout.hxx

```diff
@@ -17,6 +17,7 @@
 
 #include "TMVA/DNN/Architectures/Reference.h"
 #include "TRandom.h"
+#include <time.h>
```

Comment:
Can this part be removed once @lmoneta accepts https://github.com/root-project/root/pull/8674 ?

Reply:
@Axel-Naumann Sure! Especially that apparently it has a side effect... 🤔